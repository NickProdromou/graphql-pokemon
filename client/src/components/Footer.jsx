import React from "react";
import Styled from "styled-components";
import { Page, Row, Column } from "hedron";
import { spacing, colours } from "../style/variables";
import { type } from "../style/mixins/index";

const Footer = () => (
  <StyledFooter className="Footer">
    <Page>
      <Row>
        <Column>
          <h1 className="FooterText">&copy; Nick Prodromou 2018</h1>
        </Column>
      </Row>
    </Page>
  </StyledFooter>
);

const StyledFooter = Styled.footer`
  background: ${colours.dividerColor};

  .FooterText {
    ${type("ui")};
    padding: 0 ${spacing.mid.level1}; 
    color: ${colours.primaryTextColor};
  }
`;

export default Footer;
