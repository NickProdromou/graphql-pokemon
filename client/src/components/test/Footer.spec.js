import React from "react";
import renderer from "react-test-renderer";
import Footer from "../Footer";

describe("Footer Component", () => {
  test("It renders without error", () => {
    const component = renderer.create(<Footer />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
