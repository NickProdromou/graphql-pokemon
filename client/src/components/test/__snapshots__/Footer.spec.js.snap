// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders without error 1`] = `
<footer
  className="Footer sc-ifAKCX gFPhJs"
>
  <div
    className="sc-bxivhb bIAnAi"
  >
    <section
      className="sc-bwzfXH irrVft"
      fluid={true}
    >
      <div
        className="sc-htpNat eWAgNU"
      >
        <h1
          className="FooterText"
        >
          Â© Nick Prodromou 2018
        </h1>
      </div>
    </section>
  </div>
</footer>
`;
